languages/elixir/exercises/concept/streams/.docs/instructions.md

```diff
@@ -0,0 +1,36 @@
+As a child, you loved playing Monopoly. While as an adult you might prefer board games with more strategy and less luck involved, you decided to dust off the rulebook and have fun implementing the game in Elixir. You start with the dice rolls.
```

Comment:
Is this maybe too pretentious? As a fan of heavy strategic board games, I find Monopoly to be one of the worst board games even invented, and I find it hard to write anything nice about it... like implying the student might be a fan of Monopoly right now 😁.

Reply:
I feel similarly about Monopoly (because it is a game that is designed to infuriate its players), but I think the story could focus more on rolling the dice.

```suggestion
While planning a games night with friends, the group decides they will revisit their childhood favorite board game: Monopoly.  While not your favorite, you decide to improve the experience by implementing the game in Elixir.  You decide that the first task to implement a robust dice rolling function.
```